style.css

```diff
@@ -539,35 +540,41 @@ section {
 
 .popup {
   position: fixed;
-  top: 50%;
+  top: 49%;
   left: 50%;
   transform: translate(-50%, -50%);
```

Comment:
Great job with your desktop-size popup! However, I noticed a few differences between your design and the provided Figma. Could you kindly address these to make your project look even more aligned and professional? Here are the points to focus on:

- [ ] In the CANOPY section, for both the back end dev and 2015, it seems that you're not using the correct button icon. Could you please add the appropriate icon? The same adjustment is needed in the mobile version.

- [ ] It appears that the width of your image is slightly larger than the provided template. Could you adjust it to match the template's dimensions?

- [ ] Additionally, please review the position of your buttons and small elements. They seem to be slightly off compared to the template. Take a look at the template and align them accordingly, please.

   ### Your design:
   ![image](https://github.com/josman227/portfolio.github.io/assets/107952724/79896847-36b3-42e3-8051-0dd9a9fb32f2)

   ### Figma design:
   ![image](https://github.com/josman227/portfolio.github.io/assets/107952724/cf7740fd-3b8a-4a6c-967c-a6e03a5a17bc)